### ISSUE TITLE
Heavily annotation dependent code not work, for example retrofit.

### ISSUE THREAD

**Issue by xyhuangjinfu** [pilgun/acvtool#2]
My app use retrofit to send http request, apk can't work after instrumented.

Error message below:
"No Retrofit annotation found (parameter #1) for method ....."

**Comment by pilgun** [comment]
Yep, there is a problem with retrofit. I am going to solve it soon.
